### ISSUE TITLE
Buttons not clickable in all locations

### ISSUE THREAD

**Issue by brtwrst** [almarklein/timetagger#242]
Hello,

i just started up the docker version and noticed that the buttons have some "dead area" where clicking will do nothing.
I tried it on Desktop (Brave + Firefox) and Mobile (Brave) and in all 3 cases i notice the same thing.

Here is a bad illustration of the "dead area"
![image](https://user-images.githubusercontent.com/6873862/187651893-024de98a-a023-4900-99d2-16081b651657.png)

**Comment by almarklein** [comment]
Oh wow, I can indeed reproduce this. Weird how I have not observed this myself yet ... Thanks for reporting!

**Comment by almarklein** [comment]
The problem was that because of the way the shadow was drawn, the button's appearance was 2px too high. Fixed in #245.

I also found that it may *look* like my mouse cursor triggers the hoover sooner at the top and left side of the button, compared to the right and bottom side. But some more testing indicates that this is because my mouse cursor is black with a white border, and the actual mouse location is on the tip of the black part (MacOS).
